Contents/Code/plex_api_helper.py

```diff
@@ -87,6 +87,36 @@ def setup_plexapi():
     return plex
 
 
+def is_field_locked(item, field_name):
```

Comment:
We can possibly remove this. I was asked by plexapi maintainers to put this into a PR there. https://github.com/pkkid/python-plexapi/pull/1293

Reply:
This is now in python-plexapi, python-plexapi-backport, and Themerr-plex

https://docs.lizardbyte.dev/projects/python-plexapi-backport/en/latest/modules/base.html#plexapi.base.PlexPartialObject.isLocked